{
  "name": "babel-plugin-try-catch-wrapper",
  "version": "0.1.3",
  "description": "wrap all code in try-catch for reporting js runtime error",
  "main": "lib/index.js",
  "files": [
    "lib/"
  ],
  "scripts": {
    "test": "babel src -d lib; ln -s .. node_modules/babel-plugin-try-catch-wrapper; mocha --ui tdd",
    "prepublish": "babel src -d lib",
    "dev": "babel src -d lib -w"
  },
  "author": "jacksky007 <jacksky007@qq.com>",
  "license": "MIT",
  "dependencies": {
    "babel": "^6.3.26",
    "babel-cli": "^6.4.5",
    "babel-core": "^6.4.5",
    "babel-helper-function-name": "^6.5.0",
    "babel-template": "^6.3.13"
  },
  "devDependencies": {
    "babel-helper-plugin-test-runner": "^6.5.0",
    "babel-plugin-transform-strict-mode": "^6.5.2",
    "babel-preset-es2015": "^6.6.0",
    "babel-preset-stage-0": "^6.5.0",
    "mocha": "^2.4.5"
  },
  "directories": {
    "test": "test"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/jacksky007/report-error/tree/master/babel-plugin-try-catch-wrapper"
  },
  "keywords": [
    "report error",
    "runtime error",
    "try-catch wrapper"
  ]
}
